libraries/Scheduler/src/Scheduler.cpp

```diff
@@ -64,11 +83,39 @@ static void __attribute__((naked)) __attribute__((noinline)) coopTaskStart(void)
 static void __attribute__((naked)) __attribute__((noinline)) coopDoYield(CoopTask* curTask) {
 	asm (
 		"mov   r12, sp;"
-		"stmia r0, {r4-r12, lr};"
+		/* store low registers */
```

Comment:
Shall this be put inside ifdef too, so we can use the more efficient store instruction when available (without copying the high registers into the low registers)?